health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/BulkUpload.js

```
@@ -0,0 +1,202 @@
+    ));
+  }, [fileData, fileUrl]);
+
+  return (
+    <React.Fragment>
+      {(!fileData || fileData?.length === 0) && (
+        <FileUploader
+          multiple={multiple}
+          handleChange={handleChange}
+          name="file"
+          types={fileTypes}
+          children={dragDropJSX}
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)